### ISSUE TITLE
JAWS - Autocomplete should announce invalid input when blurring

### ISSUE THREAD

**Issue by DaveBest99** [angular/components#3380]
#### Bug, feature request, or proposal:
Bug: AutoComplete has no error detection.

#### What is the expected behavior?
Moving focus out of an empty auto complete field where a value is required should either invoke an error or if a default value was initially assigned, reset the value to the default value. Also, moving focus out of an auto complete field that does not contain a valid entry should either invoke an error or if a default value was initially assigned, reset the value to the default value. 

#### What is the current behavior?
Currently there is no error detection and anything can be typed into the Edit box.

#### What is the use-case or motivation for changing an existing behavior?
Help users (particularly elderly, screen reader and cognitive users) with accurate input.

#### Is there anything else we should know?
The user can press Enter key to select a blank Edit box or an invalid entry "Ontario", with no notification. An aria-alert could be used to notify the user that the selected list entry is not valid.

**Comment by kara** [comment]
Related: https://github.com/angular/material2/issues/3416

**Comment by ryanpm40** [comment]
@kara I'm curious if there's any update on this? We disable/enable submit buttons in our application based on the status of required fields, so this is a bit of an issue and inconsistency from other material form controls. Thanks!
